chart/values.yaml

```
@@ -319,7 +319,24 @@ extraEnvFrom: ~
 # Airflow database & redis config
 data:
   # If secret names are provided, use those secrets
+  # These secrets must be created manually, eg:
+  #
+  # kind: Secret
+  # apiVersion: v1
+  # metadata:
+  #   name: custom-airflow-metadata-secret
+  # type: Opaque
+  # data:
+  #   connection: base64_encoded_connection_string
+
   metadataSecretName: ~
+  # When providing secret names and using the same database for metadata and
+  # result backend, for Airflow < 2.4.0 it is necessary to create a separate
+  # secret for result backend but with db+ prefix in connection. For example:
+  # data:
+  #   connection: base64_encoded_connection_string_with_db+_prefix
```

```suggestion
  # secret for result backend but with a db+ scheme prefix.
```

I don't think the inline example really adds much value? I also changed the wording a bit (so it reads better to me at least).